src/modules/eventFeedback/EventFeedbackPage.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="bg-gray-1 flex flex-col p-4 justify-center items-center w-full">
+    <div class="container flex flex-col">
+      <div class="font-heading text-4xl">Event Feedback</div>
+      <div class="text-gray-5">
+        {{ eventName }}
+      </div>
+      <QuestionStar class="mt-3" />
+      <div v-for="group in questionGroupData" :key="group.id" class="mt-3">
+        <div class="font-heading text-2xl">
+          Category {{ group.seq }}: {{ group.title }}
+        </div>
+        <div
+          v-for="question in group.questions"
+          :key="question.id"
+          class="mt-3"
+        >
+          <QuestionRadio
+            v-if="question.answerType === 'SCALE'"
+            :question="question.title"
+          />
+          <QuestionText
+            v-else
+            :question="question.title"
+            :placeholderText="placeholder"
+          />
+        </div>
+      </div>
+      <div class="flex mt-3 self-end">
+        <base-button class="w-18 h-4">Submit</base-button>
```

Comment:
Note: Will later need to have logic to prevent submitting (ex. at least must have rating done)